.gitignore

```diff
@@ -23,3 +23,4 @@ CMakeCache.txt
 *.log
 *.tlog
 build
+build-*/
```

Comment:
The existing "build" entry is not matching the build folder on at least one platform using the latest Git (Windows) thus UI clients show thousands of Untracked entries in the file view.  Adding "ignore = dirty" or "ignore = untracked" to the .gitmodules also does not seem to ignore those files.